### ISSUE TITLE
ClipManager script with '/bin/sh' shebang contains Bash-specific syntax

### ISSUE THREAD

**Issue by roadsidev** [JaKooLit/Debian-Hyprland#3]
**Description:**
ClipManager doesn't work

**Actual Behavior:**
When executing the ClipManager.sh script (with Bash as the default shell), it results in the following error:

```
chris@hostname:~/.config/hypr/scripts$ ./ClipManager.sh 
./ClipManager.sh: 8: [[: not found
```

This issue arises despite the script's shebang line specifying #!/bin/sh, implying POSIX shell compatibility.

**Expected Behavior:**
The ClipManager.sh script should execute without errors. 

The shebang line suggests compatibility with `/bin/sh`, but it seems to be using Bash-specific syntax for the `if` condition. See reference: https://stackoverflow.com/a/49994023

**Comment by roadsidev** [comment]
This issue is being addressed by PR #4

**Comment by JaKooLit** [comment]
Thanks alot, I think it was the same time you submitted the PR and I have already pushed correction. Thanks alot bro

https://github.com/JaKooLit/Debian-Hyprland/commit/92ac38298ed21c40e828e7bae0a93df61722c4cd
